### ISSUE TITLE
Gradient visualisation

### ISSUE THREAD

**Issue by MLdoesML** [etheleon/deepore#6]
tf.reduce_mean(grad[1]) -> tf.reduce_mean(grad[1][0]) pls change this on sat
